presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoConnection.java

```diff
@@ -724,4 +726,9 @@ private static String getIsolationLevel(int level)
         }
         throw new SQLException("Invalid transaction isolation level: " + level);
     }
+
+    private static String generateTraceToken()
```

Comment:
Generating a fake one doesn’t seem useful and could be confusing. Lets not pass one unless the user provides it.